.github/workflows/merge-gate.yml

```diff
@@ -14,33 +14,17 @@ jobs:
 
   slow-checks:
     name: Slow
-    runs-on: ubuntu-latest
-
-    # Even though the environment "manual-approval" will be created automatically,
-    # it still needs to be configured to require interactive review.
-    # See project settings on GitHub (Settings / Environments / manual-approval).
-    environment: manual-approval
-
-    # Replace the steps below with the required actions 
-    # and/or add additional jobs if required
-    # Note: 
-    #   If you add additional jobs, make sure they are added as a requirement 
-    #   to the approve-merge job's input requirements (needs).
-    steps:
-      - name: Tests
-        run: |
-          echo "Slow tests ran successfully"
-
+    uses: ./.github/workflows/slow-checks.yml
```

Comment:
This structure is a bit different than what @ckunki and I discussed with @tkilias . Let's discuss offline